pom.xml

```diff
@@ -0,0 +1,62 @@
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.hotels</groupId>
```

Comment:
not sure if this is oversight... otherwise looks good.

Reply:
Well spotted, but no, this is intentional. We have an hcom-oss parent pom that contains all the basic plugins as well as setup necessary to publish to Sonatype. We might as well re-use what's in there as it should all be generic, if we find it's not then we can move it out elsewhere.